members/admin/__init__.py

```diff
@@ -352,6 +354,20 @@ def person_age_years(self, item):
     person_age_years.short_description = "Alder"
     person_age_years.admin_order_field = "-member__person__birthday"
 
+    def person_gender(self, item):
+        if item.member.person.gender == "MA":
+            return "Dreng"
+        else:
+            return "Pige"
```

Comment:
Måske vi skal lave det som === i stedet så vi også sikrer typen er korrekt, og måske skal vi bruge elif i stedet for else - så kan vi validere på at kønnet f.eks. ikke er null eller en anden værdi.

Reply:
jeg kan ikke lige se at vi bruger === andre steder i python code.

changed to:
        if item.member.person.gender == "MA":
            return "Dreng"
        elif item.member.person.gender == "FE":
            return "Pige"
        else:
            return "andet"